promdens.py

```diff
@@ -66,39 +108,35 @@ def read_input_data(self, fname='ics.dat'):
         :return: store all the data within the class
         """
         print(f"* Reading data from file '{fname}' of input file type '{self.input_type}'.")
-        if self.input_type == 'file':
```

Comment:
I see why you removed the condition, but it's there for a very near future, because I want to implement NewtonX output reading. Thus, I would keep it there, so that I don't need to code it again in a week. But I'm fine with the else condition disappearing.

Reply:
The point of this was also that the new code should be in a separate function (and the current code preferably as well). I can put this back, but it feels like this will change anyway.

Reply:
Ok, I'm not sure what you mean exactly but the condition is not necessary now and we can discuss once we need a new file type.